widgetastic_patternfly.py

```diff
@@ -929,8 +929,9 @@ def expand_path(self, *path, **kwargs):
             if isinstance(step, basestring):
                 # To speed up the search when having a string to match, pick up items with that text
                 child_items = self.child_items_with_text(node, step)
-            else:
+            if not child_items:
                 # Otherwise we need to go through all of them.
+                # Also covers situation when step is string but @title is missing
                 child_items = self.child_items(node)
```

Comment:
In this case template_name present only as text under locator but not as "@title"